spec.md

```diff
@@ -1802,6 +1802,9 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 A Controller plugin MUST implement this RPC call if plugin has `EXPAND_VOLUME` controller capability.
 This RPC allows the CO to expand the size of a volume.
 
+This operation MUST be idempotent.
+If a volume corresponding to the specified volume ID is larger than or equal to the capacity requirements after expansion, the plugin SHOULD reply 0 OK.
+
```

Comment:
Should this say "after expansion" or "before expansion"?

Reply:
"The capacity requirement after expansion" means target size. The sentence means to compare current volume size with target size. If current size is larger than or equal to target size, the interface SHOULD reply 0 OK.